src/assets/locales/hi.js

```diff
@@ -114,6 +114,11 @@ export default {
           done: "आगे बढ़ें",
           delete: "हटा दें",
         },
+        size_info_text: {
+          info: "फाइल का साइज ज़्यादा से ज़्यादा 10 MB हो सकता है",
```

Comment:
why not run this through Google translate? "अधिकतम छवि आकार की अनुमति 10 एमबी . है" 
We have not written english words in hindi unless absolutely necessary right?